### PR TITLE
✨ Add Get and Update to Site API

### PR DESCRIPTION
**Pull request by mikedijkstra**
This PR will:

- Add `Site.get`
- Add `Site.update`
- Add examples for getting a Site
- Add examples for updating a Site

### CODE REVIEW

File: examples/nodejs/sites/update.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const { Site } = require('calibre')
+
+const updateSiteSettings = async () => {
+  const slug = 'calibre'
+  const name = 'Calibre Marketing Site'
+
+  // Update the site settings
```
Comment (benschwarz) [inline examples/nodejs/sites/update.js:9]: Is this what it does?

```suggestion
  // Rename the site
```

**Review by benschwarz — CHANGES_REQUESTED** [review]
Could you add these examples to the `examples/nodejs/README.md`?

**Comment by mikedijkstra** [comment]
> Could you add these examples to the `examples/nodejs/README.md`?

Updated 👍

**Review by benschwarz — APPROVED** [review]
LGTM!

**Pull request merged**